### PR TITLE
🐛 fix(calendars): show read only calendars from delegateSources

### PR DESCRIPTION
**Pull request by jgoldhammer**
- after initialising the eventstore with the delegateSources a list of all calendars are displayed

### Status
**IN DEVELOPMENT**

### Description
The change shows the readonly calendars from the delegates in the macos calendar as well.

### Steps to Test or Reproduce
- create a delegate in macos calendar
- check the calendar tab in meetingbar that the calendar shows up

### CODE REVIEW

File: MeetingBar/Extensions/EventStore.swift [hunk 1]
```diff
@@ -45,7 +45,7 @@ extension EKEventStore {
                 if Defaults[.allDayEvents] == AlldayEventsAppereance.show {
                     addEvent = true
                 } else if Defaults[.allDayEvents] == AlldayEventsAppereance.show_with_meeting_link_only {
-                    let result = getMeetingLink(calendarEvent)
+                    let result = getMeetingLink(calendarEvent, acceptAnyLink: false)
```
Comment (leits) [inline MeetingBar/Extensions/EventStore.swift:48]: Looks like changes from another PR

We definitely need more tests! 😄

**Comment by leits** [comment]
LGTM
Thanks!

File: MeetingBar/Extensions/EventStore.swift (hunk 1 above)
Comment (leits) [inline MeetingBar/Extensions/EventStore.swift:48]:
```suggestion
                    let result = getMeetingLink(calendarEvent)
```

**Review by leits — APPROVED** [review]

**Comment by ckylape** [comment]
Is there a rough estimate when this might get wrapped into a release?

**Pull request merged**